### PR TITLE
New Example - Led Strip CPU Boost Display

### PR DESCRIPTION
**Pull request by ElSrJuez**
Playing around with perfmon counters and OpenRGB.

Questions and feedback are welcome!

### CODE REVIEW

**Review by jath03 — COMMENTED** [review]
Overall, I like this idea, but is too similar to the temperature example I wrote.  I added a lot of comments and edits on this file for your reference, but I could accomplish the same effect using the (currently commented out) RAM monitoring section of the temperature example.  It would only take about 5 changed lines for it to work on a different zone and with cpu usage instead of ram usage.  I'd probably accept a pr modifying the other example to be more generalized and user-friendly, but this one just seems too similar.

**Comment by jath03** [comment]
I saw your post on reddit and this looks great in action!

**Comment by ElSrJuez** [comment]
> Overall, I like this idea, but is too similar to the temperature example I wrote. I added a lot of comments and edits on this file for your reference, but I could accomplish the same effect using the (currently commented out) RAM monitoring section of the temperature example. It would only take about 5 changed lines for it to work on a different zone and with cpu usage instead of ram usage. I'd probably accept a pr modifying the other example to be more generalized and user-friendly, but this one just seems too similar.

**Comment by ElSrJuez** [comment]
HI, yes - I agree it doesnt offer a lot different.
Thanks for the awesome library!

File: examples/motherboard/motherboard.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+print(mobo)
+mobo.set_mode('direct')
+mobo.zones[zonenumber].resize(ledstripsize)
+print(mobo.zones[zonenumber])
+
+myLeds = mobo.zones[zonenumber].leds
+for i in myLeds:
+    i.set_color(red)
+
+step = ((maxperf-minperf)/len(myLeds))
```
[the remark this answers is not in the document]
Comment (ElSrJuez) [reply examples/motherboard/motherboard.py:36]: Thanks, I just noticed there may be a bug in these formulas with the zero.

File: examples/motherboard/motherboard.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# This Example grabs perfmon counters (Processor Performance %) as an indicator of CPU Boost
+# and updates via direct ARGB to a led strip
+# Sample uses Aorus Elite Wifi with Lian Li Lancool II Casae on the "Digital Bottom" 3-pin ARGB output
+
+from openrgb import OpenRGBClient
+from openrgb.utils import DeviceType
+from openrgb.utils import RGBColor
+#import wmi
+import winstats
```
[the remark this answers is not in the document]
Comment (ElSrJuez) [reply examples/motherboard/motherboard.py:10]: The perfmon counters were not returning the most accurate Boost values anyway... will take a look, thx.

**Pull request closed without merging**